server/server.go

```diff
@@ -970,3 +970,43 @@ func (s *Server) KillNonFlashbackClusterConn() {
 		s.Kill(id, false)
 	}
 }
+
+// GetMinStartTS implements SessionManager interface.
+func (s *Server) GetMinStartTS(lowerBound uint64) (ts uint64) {
+	// sys processes
```

Comment:
We may not pass this parameter `lowerBound` and get the min TS from all sessions. You already judge the TS returned with  `startTSLowerLimit` at https://github.com/pingcap/tidb/pull/39950/files#diff-d19169611c4613e5d3f09dc49da1c5d99bc6c7adb90a4b2ebbd75ac869806f50R705

Reply:
No, it should the minimum start-ts that greater than lowerBound, not the minimum start-ts of all.

Reply:
got it